api/specs/web-server/_catalog.py

```diff
@@ -130,3 +131,15 @@ def get_service_resources(
     _params: Annotated[ServicePathParams, Depends()],
 ):
     ...
+
+
+@router.get(
+    "/catalog/services/{service_key:path}/{service_version}/pricing-plan",
```

Comment:
MINOR: we used to add `versions` as as sub-resource/collection , i.e. "/catalog/services/{service_key:path}/versions/{service_version}/pricing-plan"

this way you can e.g. list all versions of a service. 

NOTE: i have just noticed that we do the same in the web-api ... I propose to improve this in the ftuure.

Reply:
Yes, I have done it in the same way as it is in the webserver API for consistency. When we decide we will change it we can change it everywhere :+1: